# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/svg/SVGGeometryProperty.h
# Commit: 8a94c155e35c
# Full Hash: 8a94c155e35c69a524e0ecb17674bb5afa9c5ad3
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-09-12 16:30:15
# Regressor Bug: 1287054
# File Overlap Count: 1
# Description:
#   Bug 1287054 part 1 - support rem, ch, ic and cap units for non-CSS lengths r=emilio,jgilbert
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D180618
# ==============================================================================

diff -r cd564dee524a -r 8a94c155e35c dom/svg/SVGGeometryProperty.h
--- a/dom/svg/SVGGeometryProperty.h	Mon Sep 11 23:42:37 2023 +0000
+++ b/dom/svg/SVGGeometryProperty.h	Mon Sep 11 23:50:11 2023 +0000
@@ -227,7 +227,10 @@
 }
 
 template <class Func>
-bool DoForComputedStyle(const SVGElement* aElement, Func aFunc) {
+bool DoForComputedStyle(const Element* aElement, Func aFunc) {
+  if (!aElement) {
+    return false;
+  }
   if (const nsIFrame* f = aElement->GetPrimaryFrame()) {
     aFunc(f->Style());
     return true;